src/styles/index.scss

```
@@ -1,3 +1,135 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+article {
+  display: inline-block;
+}
+
+.main-container {
+  width: 166px;
+  padding: 32px 15px 16px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  border: 1px solid #fff;
+  border-radius: 5px;
+}
+
+.main-container__product-img {
+  max-height: 134px;
+  max-width: 160px;
+  display: flex;
+  margin-bottom: 40px;
+}
```

Please use nesting here and in other places

```suggestion
.main-container {
  width: 166px;
  padding: 32px 15px 16px;
  display: flex;
  flex-direction: column;
  text-align: center;
  align-items: center;
  border: 1px solid #fff;
  border-radius: 5px;


  .&__product-img {
    max-height: 134px;
    max-width: 160px;
    display: flex;
    margin-bottom: 40px;
  }
}
```

---

Please, solve this comment.